### PR TITLE
add completion dialog and option

### LINKED ISSUE #77: Returning to a complete Annotation
Make it so it returns to the beginning if the user has completed all of the annotations.

Have a banner that shows that they have completed this dataset and that they are going through it again.

Modal dialog is used to accept that they want to continue and re-annotate.

Use the prompt tool to open a warning message stating that they would be re-annotating.

### PR DESCRIPTION
**Pull request by BryonLewis**
resolves #77 

Image is shown on completion of loading:
image

On dismissal, you get this screen, or after the last submit action:
image

### CODE REVIEW

File: client/src/components/UMDAnnotation.vue [hunk 1]
```diff
@@ -207,7 +213,24 @@ export default defineComponent({
           }
         }
       }
-      handler.trackSelect(maxId + 1, false);
+      const track = cameraStore.getAnyPossibleTrack(maxId + 1);
+      if (track === undefined) {
+        // We are at max segment
+        const text = 'This Video has already been annotated.  Click OK to review annotations.';
+        const res = await prompt({
+          title: 'Data already annotated',
```
Comment (waxlamp) [inline client/src/components/UMDAnnotation.vue:221]:
```suggestion
          title: 'Video already annotated!',
```

File: client/src/components/UMDAnnotation.vue [hunk 2]
```diff
@@ -207,7 +213,24 @@ export default defineComponent({
           }
         }
       }
-      handler.trackSelect(maxId + 1, false);
+      const track = cameraStore.getAnyPossibleTrack(maxId + 1);
+      if (track === undefined) {
+        // We are at max segment
+        const text = 'This Video has already been annotated.  Click OK to review annotations.';
```
Comment (waxlamp) [inline client/src/components/UMDAnnotation.vue:219]:
```suggestion
        const text = 'This video has already been fully annotated. You can choose to edit or re-do these annotations, but you should do so with caution, and generally only if you have been instructed to.';
```

File: client/src/components/UMDAnnotation.vue [hunk 3]
```diff
@@ -207,7 +213,24 @@ export default defineComponent({
...
+        const res = await prompt({
+          title: 'Data already annotated',
+          text,
+          positiveButton: 'OK',
```
Comment (waxlamp) [inline client/src/components/UMDAnnotation.vue:223]:
```suggestion
          positiveButton: 'Edit anyway',
```

**Review by waxlamp — CHANGES_REQUESTED** [review]
I've strengthened and explicitized the language to provide more of a warning feeling. Together with making the dialog a warning dialog, this should bring enough awareness to annotators to prevent issues.

**Comment by BryonLewis** [comment]
image

**Review by waxlamp — APPROVED** [review]

**Pull request merged**